src/TemplateBinding.js

```diff
@@ -1058,6 +1106,20 @@
     updateValue: function(value) {
       if (!this.deps.repeat)
         value = [value];
+      else if (Object.prototype.toString.call(value)  === '[object Object]') {
```

Comment:
right, but what does "an object" mean? :)

It doesn't seem great to rely on Object.prototype.toString here ... I'm wondering if there's a more conventional way to express the check. e.g.

```js
if (value != null && typeof value == 'object') { ... }
```